### ISSUE TITLE
UI to edit and/or delete a vendor

### ISSUE THREAD

**Issue by tngraf** [eclipse/sw360#321]
It shall be possible to edit an exisiting vendor entry (see also #114) for clearing admins.
It shall also possible to remove a vendor from a component (example: "Open Source Software" or "Nuget" are no valid vendors and should get removed.

**Comment by mcjaeger** [comment]
first issue is an ops topic

@tngraf for the second: it is about having it wiping for all releases of a component at a single attempt?

**Comment by mcjaeger** [comment]
related with https://github.com/eclipse/sw360/issues/267

**Comment by tngraf** [comment]
Yes, having it wiping for all releases of a component at a single attempt would be great,
but doing it release by release would be sufficient to solve the basic problem with wrong vendors.

**Comment by mcjaeger** [comment]
it is there for sw360 admins with the new lar file, but should be maybe for clearing admins, indeed.
